src/civis/response.py

```diff
@@ -231,9 +268,20 @@ def __setstate__(self, state):
         """Set the state when unpickling, to avoid RecursionError."""
         self.__dict__ = state
 
-    def _replace(self, key, value):
-        """Only used within this repo; `key` assumed to be in snake_case."""
-        self._data_snake[key] = value
```

Comment:
Dropping this internal method and refactoring this codebase to use `.json()` instead